sample_templates/cloudfront.yml.j2

```diff
@@ -0,0 +1,34 @@
+{#- config template for CloudFront -#}
+{#- http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cf-metricscollected.html -#}
+{#- Units are currently being reported as None for Requests, BytesDownloaded, BytesUploaded -#}
+
+{%- set metrics = {
+    'Requests':        {'stat': 'Sum', 'unit': None},
```

Comment:
It is not necessary to specify the unit in the request; it will always be in the response (it should still be in the formatter, though).